07performance/sec02Memory.md

```diff
@@ -106,61 +105,64 @@ void Transpose(vector<vector<float>> &A, vector<vector<float>> &B)
     }
 }
 ```
+
+We'll assume that our matrices are in row major order, so rows in each matrix are contiguous in memory, and we will be focusing just on reading the data from the source matrix, and ignoring writing the operations to the output matrix, since the output matrix will be filled in order so that part of the algorithm is already cache efficient. (If they were in column major order the logic would be the same except exchanging write for read: reading the source matrix would be cache efficient, but writing the output matrix would be inefficient.)

```

Comment:
I think there's been a duplicate of this paragraph somehow?

Reply:
Oh woops, dind't fix the merge conflict properly. Should be fixed now!